blog/tidal_profile_ndr_brt.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Tidal Profile - ndr_brt
+---
+
+| Tidal Cyclist  |     |
```

Comment:
I suggest putting ndr_brt on the Tidal Cyclist table heading line and removing the aka table row. It seems not needed to have both rows.